clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/RegionScopedTitusClient.java

```diff
@@ -226,17 +227,22 @@ public void terminateTasksAndShrink(TerminateTasksAndShrinkJobRequest terminateT
   }
 
   private void killTaskWithRetry(String id, TerminateTasksAndShrinkJobRequest terminateTasksAndShrinkJob) {
-    try {
-      TitusClientAuthenticationUtil.attachCaller(grpcBlockingStub).killTask(TaskKillRequest.newBuilder().setTaskId(id).setShrink(terminateTasksAndShrinkJob.isShrink()).build());
-    } catch (io.grpc.StatusRuntimeException e) {
-      if (e.getStatus() == Status.NOT_FOUND) {
-        log.warn("Titus task {} not found, continuing with terminate tasks and shrink job request.", id);
-      } else {
-        retrySupport.retry(() ->
-            TitusClientAuthenticationUtil.attachCaller(grpcBlockingStub).killTask(TaskKillRequest.newBuilder().setTaskId(id).setShrink(terminateTasksAndShrinkJob.isShrink()).build())
-          , 2, 1000, false);
-      }
-    }
+      retrySupport.retry(() -> {
+        try {
+          return TitusClientAuthenticationUtil.attachCaller(grpcBlockingStub).killTask(
```

Comment:
no return statement needed

Reply:
@tomaslin If I don't add both returns `retrySupport` yells at me, because the function it runs is supposed to return a value. Worth me changing the interface to add support for a function w/o a return type, or just keeping the return both places?

Reply:
that's annoying. Yeah, either leave as it or change the signature works. LGTM otherwise